feedback.py

```diff
@@ -0,0 +1,25 @@
+class Feedback:
```

Comment:
I cannot see multi-inheritance in the code

Reply:
Okay, I will correct all problems and make 3rd PR)